benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java

```diff
@@ -0,0 +1,267 @@
+package io.grpc.benchmarks.netty;
+
+import com.google.common.util.concurrent.MoreExecutors;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.Level;
+import org.openjdk.jmh.annotations.Param;
+import org.openjdk.jmh.annotations.Scope;
+import org.openjdk.jmh.annotations.Setup;
+import org.openjdk.jmh.annotations.State;
+import org.openjdk.jmh.annotations.TearDown;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.OutputStream;
+import java.net.InetSocketAddress;
+import java.net.ServerSocket;
+import java.net.SocketAddress;
+import java.util.concurrent.TimeUnit;
+
+import javax.annotation.Nullable;
+
+import io.grpc.Call;
```

Comment:
It was:
```
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:26: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:27: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:28: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:29: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:30: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:31: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:32: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:33: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:34: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:35: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:36: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:37: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:38: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:39: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:40: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:41: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:42: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:43: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:44: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:45: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:46: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:47: Import statement is in the wrong order. Should be in the 'THIRD_PARTY_PACKAGE' group.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:98:3: Missing a Javadoc comment.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:221: 'try' child have incorrect indentation level 9, expected level should be 8.
[ant:checkstyle] /home/ejona/clients/grpc-java/benchmarks/src/jmh/java/io/grpc/benchmarks/netty/EndToEnd.java:222: 'try' child have incorrect indentation level 9, expected level should be 8.
[ant:checkstyle] /home/ejona/clients/grpc-java/netty/src/main/java/io/grpc/transport/netty/NettyClientTransport.java:42:8: Unused import - io.grpc.MethodType.
```

Checkstyle doesn't fail the build, currently (except on Travis, or if you enable the option). If you do a build, ignore the warnings, and then do subsequent builds for different files, the checkstyle will be cached and not re-printed.